proto/xion/dkim/v1/state.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
minor nit, usually you name the main proto file for the module as its name, for instance "dkim.proto"

Reply:
was following the recommendation [here](https://github.com/cosmos/cosmos-sdk/blob/main/orm/README.md#defining-tables)

Reply:
Gotcha, makes sense.